v2/stats/metrics.proto

```diff
@@ -56,6 +56,15 @@ message MemoryStat {
 	uint64 pglazyfreed = 29;
 	uint64 thp_fault_alloc = 30;
 	uint64 thp_collapse_alloc = 31;
+	MemoryEntry usage = 32;
```

Comment:
wondering this should be = 1

Reply:
What do you mean?

Reply:
no

Reply:
instead of a struct, can we just add these as field? keep the stats flat and simple

Reply:
Sure